sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj

```diff
@@ -6,12 +6,18 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.CosmosDB</PackageId>
     <Description>Provides developers with libraries for the CosmosDB under Azure Resource manager to manage CosmosDB Account, Databases and child resources and their available management capabilities. Create, Delete, Update CosmosDB Account, Databases and child resources. Note: This client library is for CosmosDB under Azure Resource Manager.</Description>
-    <Version>1.3.2-preview</Version>
+    <Version>1.3.3-preview</Version>
     <AssemblyName>Microsoft.Azure.Management.CosmosDB</AssemblyName>
     <PackageTags>management;cosmosdb;</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        Adding a fix for long running operations returning Unauthorized.
+        Adding new APIs for Point in Time Restore feature
```

Comment:
Is PITR the official public name for this feature? this changelog would be available publicly so let us make sure it is as informative as possible

Reply:
yes.  we are calling the feature continuous backup, but in this instance, the apis apply to restore scenario so calling it PITR would make more sense.  we are using continuous backup/pitr in the public doc.